client/webserver/webserver.go

```diff
@@ -284,7 +284,14 @@ func New(cfg *Config) (*WebServer, error) {
 
 	// Middleware
 	if log.Level() == dex.LevelTrace {
-		mux.Use(middleware.Logger)
+		mux.Use(middleware.RequestLogger(&middleware.DefaultLogFormatter{
+			Logger: &chiLogger{
+				// Using stdout for these logs because while colored output looks fine in
+				// console, it's inconvenient to deal with in a log file (e.g. hard to grep).
+				Logger: dex.StdOutLogger("WEB", dex.LevelTrace),
```

Comment:
Oh, if running with `--loglocal` the timezone discrepancy is back anyway. (see the final arg in this variadic constructor)
Also note that on simnet `--loglocal` is ignored, presently, but we can remove that override in dexc's `runCore`.  I don't know why we did that and it's confusing for changing this.
I'm open to switching to no-color in the same log file, since the alternative appears to be propagating the `utc bool` to the webserver.